cl/corpus_importer/management/commands/normalize_judges_opinions.py

```diff
@@ -0,0 +1,79 @@
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.people_db.lookup_utils import (
+    lookup_judge_by_last_name,
+    lookup_judges_by_last_name_list,
+)
+from cl.search.models import Opinion, OpinionCluster
+
+
+class Command(VerboseCommand):
+    help = (
+        "Populate author field in opinions using author_str field and "
+        "populate panel in opinion clusters using judges field. "
+    )
+
+    def handle(self, *args, **options):
+
+        self.normalize_authors_in_opinions()
+
+        self.normalize_panel_in_opinioncluster()
+
+    def normalize_authors_in_opinions(self):
+        """Normalize author_str in opinions"""
+
+        # Get opinions that have no author object and have author_str
+        opinions_with_author_str = Opinion.objects.exclude(
+            author__isnull=False
+        ).exclude(author_str__exact="")
+        for opinion in opinions_with_author_str:
+            date_filed = opinion.cluster.docket.date_filed
+            court_id = opinion.cluster.docket.court.pk
```

Comment:
This will avoid a query by getting the ID from the docket table:

```suggestion
            court_id = opinion.cluster.docket.court_id
```